src/main/java/seedu/address/model/menu/MenuItem.java

```diff
@@ -96,11 +93,7 @@ public int hashCode() {
     @Override
     public String toString() {
         final StringBuilder builder = new StringBuilder();
-        builder.append(getName())
-                .append(" Code: ")
-                .append(getCode())
-                .append(" Price: ")
-                .append(getPrice());
+        builder.append(getName()).append(" Code: ").append(getCode()).append(" Price: ").append(getPrice());
```

Comment:
Even though this edit does not violate any conventions, I believe the previous form was more readable